File: OpenCredentialPublisher.Data/Models/LogModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace OpenCredentialPublisher.Data.Models
{
    [Table("Logs")]
    public class LogModel
    {
        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        public string Message { get; set; }
        public string MessageTemplate { get; set; }
        [MaxLength(128)]
        public string Level { get; set; }
        public DateTimeOffset Timestamp { get; set; }
        public string Exception { get; set; }
        [Column(TypeName = "xml")]
        public string Properties { get; set; }
        public string LogEvent { get; set; }
    }
}

File: OpenCredentialPublisher.Wallet/src/ClientApp/src/app/shared/models/LogModel.ts
export class logModel {
  id: number;
  message: string;
  messageTemplate: string;
  level: string;
  timestamp: Date;
  exception: string;
  properties: string;
  logEvent: string;

  constructor() {
    this.id = -1;
    this.message = '';
    this.messageTemplate = '';
    this.level = '';
    this.timestamp: Date;
    this.exception = '';
    this.properties = '';
    this.logEvent = '';
  }

}
